src/components/ElevatedButton/index.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import ElevatedButton from '.';
+
+describe('ElevatedButton', () => {
+  const label = 'Sign in';
+  const dataTestId = 'sign-in_button';
+  it('renders a full-width button', () => {
+    render(
+      <ElevatedButton isFullSize data-test-id={dataTestId}>
+        {label}
+      </ElevatedButton>
+    );
+
+    const button = screen.getByTestId(dataTestId);
+
+    expect(button).toBeVisible();
+    expect(button).toHaveTextContent(label);
+    expect(button).toHaveAttribute('type', 'button');
+    expect(button).toHaveClass('w-full');
+  });
+
+  it('renders a wrap-content button', () => {
```

Comment:
we could simply say that

```suggestion
  it('does NOT render a full-width button', () => {
```

It makes it easier to read, what do you think?